src/TSHScoreboardWidget.py

```diff
@@ -436,7 +436,7 @@ def GenerateThumbnail(self):
             thumbnailPath = thumbnail.generate(settingsManager=SettingsManager, scoreboardNumber=self.scoreboardNumber)
             msgBox.setText(QApplication.translate(
                 "thumb_app", "The thumbnail has been generated here:") + " ")
-            msgBox.setIcon(QMessageBox.NoIcon)
+            msgBox.setIcon(QMessageBox.Information)
```

Comment:
Wait, isn't that overriding #603 ?

Reply:
Hmmmm, not sure why that was reverted even though it was merged in properly 🤔 either way, I made a manual commit for this, so we should be good now.